spec/sanity/blueprints.js

```diff
@@ -13,23 +14,24 @@ describe('blueprints page', function(){
         components.ui.LoginPage.goTo().login('admin', 'admin');
     });
 
-    describe('blueprint view', function(){
+    describe('blueprint view', function() {
 
-        beforeEach(function(){
-            components.ui.blueprints.IndexPage.goToBlueprint({'name' : testConf.blueprints.blueprintToRead });
+        beforeEach(function () {
+            components.ui.blueprints.IndexPage.goToBlueprint({'name': testConf.blueprints.blueprintToRead});
         });
 
         it('should have all sections', function (done) {
             components.ui.blueprints.BlueprintPage.goToNodes();
             components.ui.blueprints.BlueprintPage.goToSource();
-            browser.sleep(1000).then( done );
+            browser.sleep(1000).then(done);
         });
+    });
 
-        it('should have working source section', function(done) {
+    describe('source view', function(){
 
-            logger.trace('start blueprint source section test');
+        it('should have working source section', function(done) {
+            components.ui.blueprints.IndexPage.goToBlueprint({'name': testConf.blueprints.blueprintToRead});
             BlueprintPage.goToSource();
-            expect(BlueprintPage.Source.getLoadingMessage()).toBe('Generating Blueprint Source View...');
```

Comment:
There is an implicit expect under 'getFileTitle'.. it should externalized to the test for readability. we sometimes assert stuff during internal functions like 'goToBlueprint' we expect to have such a blueprint, to make debug easier when test fails. but in this scenario it feels forced.